server/config/app-template.js

```diff
@@ -1,5 +1,7 @@
+import { WP_URL as APP_WP_URL } from '../../app/config/app.js'
+
 // Wordpress
-export const WP_URL = 'http://localhost/starward_wp';
+export const WP_URL = APP_WP_URL;
```

Comment:
The app needs WP_URL and cannot reference a /server/ file, thus we move it to the /app/config/app.js